packages/crypto/src/lib/crypto.ts

```diff
@@ -359,14 +361,45 @@ function base64ToBufferAsync(base64) {
     });
 }
 
-async function getAmdCert(url: string) {
-  // unfortunately, until AMD enables CORS, we have to use a proxy when in the browser
-  // This project is hosted on heroku and uses this codebase: https://github.com/LIT-Protocol/cors-proxy-amd
-  if (isBrowser()) {
-    // CORS proxy url
-    url = `https://cors.litgateway.com/${url}`;
+/**
+ * Asynchronously fetches an AMD certification from a specified URL using a CORS proxy.
+ * This approach is used to bypass CORS restrictions when accessing external APIs that do not support CORS.
+ * It attempts to fetch the AMD cert through a proxy and if the proxy fetch fails, it throws an error.
+ *
+ * @param url The URL from which to fetch the AMD cert.
+ * @returns A Promise that resolves to a Uint8Array containing the AMD certification data.
+ * @throws An error detailing HTTP or network issues encountered during the fetch process.
+ */
+async function getAmdCert(url: string): Promise<Uint8Array> {
+  const proxyUrl = `${LIT_CORS_PROXY}/${url}`;
+
+  log(
+    `[getAmdCert] Fetching AMD cert using proxy URL ${proxyUrl} due to CORS restrictions.`
```

Comment:
This is no longer only used to bypass CORS, so if we're gonna log and JSDoc this, we should mention that we always call it to avoid being rate limited by AMD in these logs and update the JSDoc content to describe that we now _always_ use this proxy to bypass rate limiting